bingo/bingo-elastic/python/bingo_elastic/model/record.py

```diff
@@ -29,37 +30,50 @@ def __set__(self, instance: IndigoRecord, value: Dict):
 
 class WithIndigoObject:
     def __set__(self, instance: IndigoRecord, value: IndigoObject) -> None:
+        value.aromatize()
```

Comment:
We can accidentially affect the state of the input object, probably it's better to clone it first.

Reply:
Fixed!